src/components/UpdateAppModal/UpdateAppModalPropTypes.js

```diff
@@ -1,8 +1,11 @@
 import PropTypes from 'prop-types';
+import {withLocalizePropTypes} from '../withLocalize';
 
 const propTypes = {
     // Callback to fire when we want to trigger the update.
     onSubmit: PropTypes.func,
+
+    ...withLocalizePropTypes,
```

Comment:
not sure I understand why this is necessary.

Reply:
nvm!

Reply:
I don't understand it either